pkg/spireapi/entryapi.go

```diff
@@ -18,17 +18,32 @@ package spireapi
 
 import (
 	"context"
+	"fmt"
 
 	entryv1 "github.com/spiffe/spire-api-sdk/proto/spire/api/server/entry/v1"
+	"github.com/spiffe/spire-api-sdk/proto/spire/api/types"
 	apitypes "github.com/spiffe/spire-api-sdk/proto/spire/api/types"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
```

Comment:
nit: make a type for better compiler safety `type Field string` then use that in GetUnsupportedFields and the other code